api/v3/openapi.yaml

```diff
@@ -2061,11 +2061,15 @@ components:
     ProduceResponse:
       type: object
       required:
+        - error_code
```

Comment:
Should we add an explanation in the produce api section in `paths` that the users should *always* check the `error_code` field, and that `200` means no error?